layer/VkLayer_gfxreconstruct.exp

```diff
@@ -0,0 +1,5 @@
+_vkGetInstanceProcAddr
+_vkGetDeviceProcAddr
+_vkEnumerateInstanceLayerProperties
+_vkEnumerateInstanceExtensionProperties
+_vkNegotiateLoaderLayerInterfaceVersion
```

Comment:
Why isn't this list the same as the lists for Linux and Windows and Android?

Reply:
Those are Android-specific, however they still may be used by emulators.  So I went ahead and added them as well now.  It was really an oversight.  Thanks for catching this.